cmd/utils/flags.go

```diff
@@ -1726,6 +1731,37 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 		cfg.NetworkId = *cfg.Genesis.GetNetworkID()
 	}
 
+	if ctx.GlobalIsSet(EthProtocolsFlag.Name) {
+		protocolVersions := SplitAndTrim(ctx.GlobalString(EthProtocolsFlag.Name))
+		if len(protocolVersions) > 0 {
```

Comment:
We can flip this logic to remove a nested condition (and be a little more Go-idiomatic):

```
if len(protocolVersions) == 0
  // handle the Fatalf for bad usage

for _, versionString := range protocolVersions
```